WireAuthentication/Sources/WireAuthenticationUI/Views/Login/LoginViaEmail/LoginViaEmailViewModel.swift

```diff
@@ -61,9 +61,10 @@ package final class LoginViaEmailViewModel: ObservableObject {
     func submitPassword(_ password: String) {
         Task.detached {
             do {
-                try await self.loginViaEmailUseCase.invoke(
+                _ = try await self.loginViaEmailUseCase.invoke(
```

Comment:
suggestion: if we already have a jira ticket for completing the login flow then perhaps we can add a todo here.